packages/@aws-cdk/aws-cloudfront-origins/lib/s3-origin.ts

```diff
@@ -50,10 +56,13 @@ export class S3Origin implements cloudfront.IOrigin {
  * Contains additional logic around bucket permissions and origin access identities.
  */
 class S3BucketOrigin extends cloudfront.OriginBase {
-  private originAccessIdentity!: cloudfront.OriginAccessIdentity;
+  private originAccessIdentity!: cloudfront.OriginAccessIdentity | cloudfront.IOriginAccessIdentity;
```

Comment:
No reason to have this be a type union, since an `OriginAccessIdentity` *is* an `IOriginAccessIdentity`.

```suggestion
  private originAccessIdentity!: cloudfront.IOriginAccessIdentity;
```